index.css

```diff
@@ -0,0 +1,197 @@
+@import url(https://fonts.googleapis.com/css?family=Oswald);
+
+.container {
+  font-family: Oswald;
+}
+
+a:hover {
+  color: #9068be;
+}
+
+nav {
+  background-color: #e62739;
+}
+
+
+ul.menu-options > li {
+  display: inline-block;
+  list-style-type: none;
+  text-align: center;
+  font-style: bold;
+  
+}
+
+#startrly {
+  margin-right: 400px;
+  margin-left: 1px;
+}
+
+#try {
+  margin-left: 300px;
+  margin-right: 300px;
+}
+
+.intro {
+  display: block;
+  text-align: center;
+  background-color: #6ed3cf;
+}
+
+.intro-heading {
```

Comment:
Nice names for your classes! I think they're helpful to identifying what class you're selecting.